kolibri/core/assets/src/views/SyncStatusDescription.vue

```diff
@@ -0,0 +1,50 @@
+<template>
+
+  <p>
+    {{ syncTextDisplayMap }}
+  </p>
+
+</template>
+
+
+<script>
+
+  import { SyncStatus } from 'kolibri.coreVue.vuex.constants';
+
+  export default {
+    name: 'SyncStatusDescription',
+    props: {
+      syncStatus: {
+        type: String,
+        default: '',
+      },
+    },
+    computed: {
+      syncTextDisplayMap() {
```

Comment:
Maybe I'm too nitpicky, but seing a switch that can be so easily replaced by an object looks ugly to me:
```
const statusTranslations= {SyncStatus.RECENTLY_SYNCED:this.$tr('syncedDescription') , 
SyncStatus.QUEUED:this.$tr('queuedDescription'), ....}

return statusTranslations[this.syncStatus] || ''
```
looks better than a switch for me, but I can be wrong